content/blog/entries/cc-search-accessibility-week3-4/contents.lr

```diff
@@ -0,0 +1,98 @@
+title: Internationalization Continued: Handling strings in the store
+---
+categories:
+cc-search
+community
+gsoc
+open-source
+
+---
+author: AyanChoudhary
+---
+series: gsoc-2020-ccsearch-accessibility
+---
+pub_date: 2020-06-26
+---
+body:
+
+These are the second two weeks of my internship with CC. I am working on improving the accessibility of cc-search and internationalizing it as well.
+The internationalization work from the previous post is continued here and we also solve an interesting issue of translating strings from the store.
```

Comment:
I would make it clear that you're talking about `Vuex` here. "the store" is too vague